articles/ssl-tls/index.md

```diff
@@ -0,0 +1,75 @@
+
+Photo by [Markus Spiske](https://unsplash.com/@markusspiske?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText) on [Unsplash](https://unsplash.com/photos/iar-afB0QQw?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText)
+
+**Introduction**
+
+2020 was a challenging year that resulted in some crucial, groundbreaking innovations and changes across domains. With the pandemic compelling various operations and workforce to go remote, thereby leaving the infrastructure susceptible to increased cyber attacks, the need for security has undoubtedly been reinforced. Anti-malware softwares, intrusion prevention systems, firewalls, Virtual Private Networks (VPNs) are some of the rudimentary technologies required to safeguard your resources these days. In this article, we will focus on one such security technique i.e SSL/TLS communication encryption protocol. 
+
+In simple terms, encryption is the process of converting data into another form (referred to as Cipher text) that can be understood only by parties that possess a secret decryption key. To everybody else, it’s gibberish.  [Google's transparency report](https://transparencyreport.google.com/https/overview?hl=en) shows the significant increase in the amount of encrypted traffic on the internet over the last few years. As of December 2020, approximately 95% of all traffic across Google was encrypted. Many web browsers have been relentlessly pushing websites to use encryption by marking HTTP sites as unsafe. 
+
+**What makes HTTPS sites safe?**
+
+Hypertext Transfer Protocol Secure ([HTTPS](https://en.wikipedia.org/wiki/HTTPS)) is an extension of Hypertext Transfer Protocol (HTTP). All HTTPS traffic is encrypted using a communication protocol called Secure Sockets Layer (SSL)/ Transport Layer Security (TLS) protocol.
+
+**What is SSL/TLS?**
+
+Secure Sockets Layer (SSL) and its successor, Transport Layer Security (TLS) are encryption protocols designed to ensure security, reliability and authenticity of the information being exchanged. Although SSL is deprecated and replaced by TLS, the industry uses the two terms interchangeably. To avoid confusion, we’ll continue to refer to TLS as SSL/TLS throughout this article.
+
+**History of SSL/TLS**
+
+The earliest original SSL protocols were developed at Netscape. SSL 2.0 was the first version to be released to the public in 1995. SSL 2.0 had vulnerabilities and consequently, [SSL 3.0](https://tools.ietf.org/html/rfc6101) was developed and released in the following year. In 2014, SSL 3.0 was found to be vulnerable to the [POODLE attack](https://en.wikipedia.org/wiki/POODLE). It was deprecated in 2015. SSL 3.0 was shortly followed by TLS 1.0 in 1999. We’ve had TLS 1.1, TLS 1.2 and TLS 1.3 so far. TLS 1.0 and TLS 1.1 were deprecated in 2020. You can read more about the protocol’s history and development [here](https://en.wikipedia.org/wiki/Transport_Layer_Security#History_and_development).
```

Comment:
> POODLE Attack

It would be great if you could give a brief description of the attack and what it is in 1-2 lines